concent_api/common/helpers.py

```diff
@@ -17,6 +18,7 @@
 
 from core.exceptions                import Http400
 from common.constants                import ErrorCode
+from common.constants import ETHEREUM_PUBLIC_KEY_LENGTH
```

Comment:
Please fix the two imports above too.

I'd prefer to have the whole "block" reformatted rather than just one line because otherwise the result looks as if we had not consistent style at all.

Reply:
:+1: done